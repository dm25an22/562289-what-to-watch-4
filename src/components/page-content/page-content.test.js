import React from "react";
import renderer from "react-test-renderer";
import PageContent from "./page-content.jsx";
import {Router} from "react-router-dom";
import {history} from "../../history";

const MockComponent = () => {
  return (
    <React.Fragment>
      <div>
        <h1>Text</h1>
      </div>
    </React.Fragment>
  );
};
it(`render PageContent component with children`, () => {

  const tree = renderer.create(
      <Router history={history} >
        <PageContent>
          <MockComponent />
        </PageContent>
      </Router>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
