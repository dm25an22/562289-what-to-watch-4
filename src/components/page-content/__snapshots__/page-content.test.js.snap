// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PageContent component with children 1`] = `
<div
  className="page-content"
>
  <section
    className="catalog"
  >
    <h2
      className="catalog__title visually-hidden"
    >
      Catalog
    </h2>
    <div>
      <h1>
        Text
      </h1>
    </div>
  </section>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
